docs/home/howto/OS_PATCHING.md

```diff
@@ -100,3 +115,51 @@ Available options:
 ## Additional information
 
 Red Hat provides notifications about security flaws that affect its products in the form of security advisories. For more information, see [here](https://access.redhat.com/security/updates/advisory).
+
+### Ubuntu
+
+For automated security patches Ubuntu uses unattended-upgrade facility. By default it runs every day. To verify it on your system, execute:  
+`dpkg --list unattended-upgrades`
+`cat /etc/apt/apt.conf.d/20auto-upgrades | grep Unattended-Upgrade`
+
+For information how to change Unattended-Upgrade configuration, see [here](https://github.com/mvo5/unattended-upgrades/blob/master/README.md).
```

Comment:
Hmmmm ... I would rather reference some official documentation in our official documentation if that is possible, but that is rather minor issue.